tests/test_lib.py

```diff
@@ -1,53 +1,59 @@
 import sys
 sys.path = [".."] + sys.path
 
+import os
 import decimal
 
-from . import TEST_DIR
-from . import ccmtest
-from ccmlib.cluster import Cluster
+from tests import TEST_DIR
+from tests import ccmtest
+# from ccmlib.cluster import Cluster
```

Comment:
if we ditch cassandra test out of here, we could remove those comments

Reply:
Of course, this is why it's a draft

Reply:
lets clean it up just a bit and merge...